runtime.js

```diff
@@ -456,7 +456,7 @@ const _makeAppUrl = appId => {
   });
   return URL.createObjectURL(b);
 };
-const _loadScript = async (file, {files = null, parentUrl = null} = {}) => {
+const _loadScript = async (file, {files = null, parentUrl = null} = {}, instanceId) => {
```

Comment:
`instanceId` is feels like it should be part of the `options`.

Reply:
40f66f1